src/content.js

```diff
@@ -0,0 +1,8 @@
+const text = require('./text.js');
+
+module.exports = (req, res) => {
+  res.render('content', {
```

Comment:
can just do `res.render('content')` - no need for anything else since the lang is already set by this point